<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>458</width>
    <height>312</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>DejaVu Serif</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QDial" name="dial">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>180</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>94</width>
      <height>181</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QSpinBox" name="spinBox_da1">
       <property name="font">
        <font>
         <family>DejaVu Sans</family>
        </font>
       </property>
       <property name="maximum">
        <number>5000</number>
       </property>
       <property name="singleStep">
        <number>10</number>
       </property>
       <property name="value">
        <number>2500</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="verticalSlider_da1">
       <property name="maximum">
        <number>5000</number>
       </property>
       <property name="singleStep">
        <number>10</number>
       </property>
       <property name="value">
        <number>2500</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>94</width>
      <height>181</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QSpinBox" name="spinBox_da0">
       <property name="font">
        <font>
         <family>DejaVu Sans</family>
        </font>
       </property>
       <property name="maximum">
        <number>5000</number>
       </property>
       <property name="singleStep">
        <number>10</number>
       </property>
       <property name="value">
        <number>2500</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="verticalSlider_da0">
       <property name="maximum">
        <number>5000</number>
       </property>
       <property name="singleStep">
        <number>10</number>
       </property>
       <property name="value">
        <number>2500</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>431</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>      AD0: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_an0">
       <property name="font">
        <font>
         <family>DejaVu Sans</family>
        </font>
       </property>
       <property name="maximum">
        <number>6000</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>      AD1:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_an1">
       <property name="font">
        <font>
         <family>DejaVu Sans</family>
        </font>
       </property>
       <property name="maximum">
        <number>6000</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>      AD2:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_an2">
       <property name="font">
        <font>
         <family>DejaVu Sans</family>
        </font>
       </property>
       <property name="maximum">
        <number>6000</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QCheckBox" name="checkBox_ramp">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>70</y>
      <width>191</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Ramp-UP Enable</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>190</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_adda">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>100</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Control AD0 by DA0</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_addaBOTH">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>130</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Control AD0/1 by DA0</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>458</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>verticalSlider_da0</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_da0</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>124</x>
     <y>232</y>
    </hint>
    <hint type="destinationlabel">
     <x>58</x>
     <y>226</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>verticalSlider_da1</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_da1</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>244</x>
     <y>202</y>
    </hint>
    <hint type="destinationlabel">
     <x>190</x>
     <y>238</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
